src/outputgeneration/ParseTreeWriter.js

```diff
@@ -1137,6 +1139,7 @@ export class ParseTreeWriter extends ParseTreeVisitor {
    * @param {PropertyVariableDeclaration} tree
    */
   visitPropertyVariableDeclaration(tree) {
+    if (this.options_.outputLanguage === 'es6') return;
```

Comment:
The writers only responsibility is to serialize the AST back to source code. It should not make this kind of decisions.